management-interface.go

```diff
@@ -397,10 +436,13 @@ func addWPANetwork(ssid string, password string) error {
 	} else if strings.ToLower(ssid) == "bushnet" {
 		return errors.New("SSID cannot be bushnet")
 	}
+	if err := restartNetwork(); err != nil {
+		return err
+	}
 
 	networkExists, err := doesWPANetworkExist(ssid)
 	if err != nil {
-		return err
+		log.Println(err)
```

Comment:
Why is the error no longer returned here?

Reply:
This was due to wpa_cli not being available to used when in hotspot mode so it wasn't possible to add a new wifi network without it throwing an error. It still works in it's normal use cases